[2025-05-17T12:00:00.699+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-17T12:00:00.735+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monitor_T3.monitor_model scheduled__2025-05-17T11:00:00+00:00 [queued]>
[2025-05-17T12:00:00.744+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monitor_T3.monitor_model scheduled__2025-05-17T11:00:00+00:00 [queued]>
[2025-05-17T12:00:00.745+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-17T12:00:00.763+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): monitor_model> on 2025-05-17 11:00:00+00:00
[2025-05-17T12:00:00.771+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monitor_T3', 'monitor_model', 'scheduled__2025-05-17T11:00:00+00:00', '--job-id', '336', '--raw', '--subdir', 'DAGS_FOLDER/monitor_T3.py', '--cfg-path', '/tmp/tmpz74yzqrf']
[2025-05-17T12:00:00.774+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=7427) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-17T12:00:00.775+0000] {standard_task_runner.py:91} INFO - Job 336: Subtask monitor_model
[2025-05-17T12:00:00.775+0000] {standard_task_runner.py:63} INFO - Started process 7429 to run task
[2025-05-17T12:00:00.831+0000] {task_command.py:426} INFO - Running <TaskInstance: monitor_T3.monitor_model scheduled__2025-05-17T11:00:00+00:00 [running]> on host b91ac73f2b68
[2025-05-17T12:00:00.939+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monitor_T3' AIRFLOW_CTX_TASK_ID='monitor_model' AIRFLOW_CTX_EXECUTION_DATE='2025-05-17T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-17T11:00:00+00:00'
[2025-05-17T12:00:00.940+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-17T12:00:00.988+0000] {logging_mixin.py:188} INFO - [monitor_T3] Prediction: 0.8026717305183411, Ground Truth: 0.65, Error: 0.15267173051834104
[2025-05-17T12:00:01.007+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-17T12:00:01.008+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-17T12:00:01.018+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=monitor_T3, task_id=monitor_model, execution_date=20250517T110000, start_date=20250517T120000, end_date=20250517T120001
[2025-05-17T12:00:01.071+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-17T12:00:01.110+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-17T12:00:01.116+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
